packages/model/src/user.ts

```diff
@@ -100,4 +100,21 @@ export interface UserResponse extends Invitee {
   role: Role;
 }
 
+export interface UserPatchRequest {
```

Comment:
Couldn't this just be a partial?

Reply:
I wanted to, but it's for some reason very different that the other types, some fields aren't even of the same type, so I didn't dare and just copied it over